ffs/rpc/service.go

```diff
@@ -269,8 +269,8 @@ func (s *Service) Info(ctx context.Context, req *InfoRequest) (*InfoReply, error
 				},
 			},
 			Wallet: &WalletInfo{
-				Address: info.Wallet.Address,
-				Balance: info.Wallet.Balance,
+				// Address: info.Wallet.Address,
+				// Balance: info.Wallet.Balance,
```

Comment:
Just disabling stuff in the grpc service for now, will update once I'm confident of the underlying changes